pyomo/solvers/plugins/solvers/CPLEX.py

```diff
@@ -601,6 +618,10 @@ def process_logfile(self):
             elif len(tokens) >= 6 and tokens[0] == "MIP" and tuple(tokens[5:]) == ('no', 'integer', 'solution.'):
                 results.solver.termination_condition = TerminationCondition.noSolution
                 results.solver.termination_message = ' '.join(tokens)
+            elif len(tokens) >= 9 and tokens[0] == "MIP" and tokens[1] == "start" and tokens[7] == "objective":
+                results.solver.warm_start_objective_value = float(tokens[8][:-1])  # remove trailing full stop
+            elif len(tokens) >= 5 and tokens[0] == "Solution" and tokens[1] == "pool:" and tokens[3] in ["solution", "solutions"] and tokens[4] == "saved.":
```

Comment:
Minor: For consecutive tokens you can do `tokens[0:2] == ["Solution", "pool:"]`. Might be slightly more concise.

Reply:
done